src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -311,15 +307,15 @@ async Task<JObject> ResolveAsInstanceMember(ArraySegment<string> parts, JObject
                     }
 
                     if (hasCurrentPartNullCondition)
```

Comment:
And use an explicit condition here like `if (part[^1] is '?' or '!')`, so we don't mistakenly have `bool throwOnNullReference = parts[0][^1] != '?'; if (!throwOnNullReference)` - which would match any character other than `?`